pipenv/cli/command.py

```diff
@@ -740,8 +740,9 @@ def verify(state):
 )
 @option("--hash", is_flag=True, default=False, help="Add package hashes.")
 @option("--exclude-markers", is_flag=True, default=False, help="Exclude markers.")
+@option("--category", is_flag=False, default='', help="Only add requirement of the specified category.")
 @pass_state
-def requirements(state, dev=False, dev_only=False, hash=False, exclude_markers=False):
+def requirements(state, dev=False, dev_only=False, hash=False, exclude_markers=False, category=''):
```

Comment:
Let's use the `categories` option that we use in the other parts of the CLI for consistency, this can accept one or multiple categories.

Reply:
@matteius Updated the PR.